pkgs/by-name/li/linux-scripts/package.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, linuxHeaders # Linux source tree
+, makeWrapper
+, stdenvNoCC
+
+, binutils
+, coreutils
+, gnugrep
+
+  # decompressors for possible kernel image formats
+, bzip2
+, gzip
+, lz4
+, lzop
+, xz
+, zstd
+}:
+
+let
+  extractVmLinuxDeps = [
```

Comment:
Maybe a more generic name since there's no reason to have different paths for different scripts?  Could literally just be `path`.